spec/person_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_relative '../person'
```

Comment:
- [ ] It would be great if you test all the methods which are in the person class to follow the project requirements 👍 because you have some custome methods such as `add_rental` etc 🤔